appserver/security/webintegration/src/main/java/com/sun/web/security/RealmAdapter.java

```diff
@@ -1534,7 +1482,24 @@ public boolean invokeAuthenticateDelegate(HttpRequest request,
                         SecurityContext.getCurrent().setAdditionalPrincipal(wrapped);
                     }
                 }
+                if (requestTracing != null && requestTracing.isRequestTracingEnabled()) {
+                    RequestEvent re = new RequestEvent("BEFORE JASPIC AUTH");
```

Comment:
Mentioned as part of the meeting: do we want a naming format for request events?
I've been naming mine similar to how Mert did his, e.g. `JBatchExecutionContextEvent`